README.md

```diff
@@ -172,6 +172,13 @@ Running Anbox from a local build requires a few more things you need to know
 about. Please have a look at the ["Runtime Setup"](docs/runtime-setup.md)
 documentation.
 
+An alias will save typing later.
```

Comment:
I would prefer not to add the alias as a recommendation to the read as this is really the choice of the user. Also abbreviating "Android" as starting the appmgr activity is wrong. Can you drop this and leave just the "sudo" change from above in the PR?